webpack/ForemanTasks/Components/TasksTable/TasksTableActions.js

```diff
@@ -94,29 +98,80 @@ export const unselectRow = id => ({
   payload: id,
 });
 
-export const actionSelected = (actionType, selected, url) => async dispatch => {
+export const bulkResumeRequest = resumeTasks => {
+  const ids = resumeTasks.map(task => task.id);
+  const url = new URI('/foreman_tasks/api/tasks/bulk_resume');
+  url.setSearch('task_ids[]', ids);
+  return API.post(url);
+};
+
+export const bulkResume = (selected, currentURL) => async dispatch => {
+  const resumeTasks = selected.filter(task => task.isResumeble);
+  if (resumeTasks.length < selected.length)
+    dispatch(
+      addToast({
+        type: 'warning',
+        message: __('Not all the selected tasks can be resumed'),
+      })
+    );
+  if (resumeTasks.length) {
+    try {
+      dispatch({ type: TASKS_RESUME_REQUEST, tasks: resumeTasks });
```

Comment:
For the new actions you added - The `tasks` key isn't consumed in the reducer or anywhere else, so I don't think it's necessary.  An action with just a `type` is fine!

Reply:
Removed the `tasks` key but left the error one, thanks!